code/modules/mob/living/carbon/human/human.dm

```diff
@@ -32,8 +32,6 @@
 
 	create_reagents(330)
 
-	prev_gender = gender // Debug for plural genders

```

Comment:
It's exactly as the comment says - A debugging tool, it lets you see what their gender was before it ended up bugged.